### PR TITLE
My review page

### PR DESCRIPTION
**Pull request by HenryChen666**

### CODE REVIEW

**Review by Zayood — APPROVED** [review]
:)

File: src/app/pages/my-review/my-review.component.scss [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+mat-paginator {
+    margin-left: 110px;
+    width: 90%;
+  }
+
+html, body{
+  height: 100%;
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+  color: #222;
```
Comment (seofernando25) [inline src/app/pages/my-review/my-review.component.scss:45]: Why did you change the font here? The project root css already has that

File: src/app/pages/my-review/my-review.component.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatPaginator } from '@angular/material/paginator';
+import {MatSnackBar} from '@angular/material/snack-bar';
+import {MatTableDataSource} from '@angular/material/table';
+import {MatDialog} from '@angular/material/dialog';
+import {UpdateMyReviewComponent} from '../../pages/update-my-review/update-my-review.component';
+
+
+export interface PeriodicElement {
```
Comment (seofernando25) [inline src/app/pages/my-review/my-review.component.ts:9]: PeriodicElement? If you're ever going to copy sample code from somewhere, at least change the name of the class. :/

**Review by seofernando25 — COMMENTED** [review]
Good stuff, the only problem is that it would be better if this tab was on the user's dropdown or only when the user was logged in. Also, don't you think its more clean if we just call it "reviews"? Later on I can move the page to some user subdomain like `dcms.com/user/reviews` or `/my/reviews`

File: src/app/pages/my-review/my-review.component.scss (hunk 1 above)
Comment (HenryChen666) [reply src/app/pages/my-review/my-review.component.scss:45]: Did not know you included that, fixed.

File: src/app/pages/my-review/my-review.component.ts (hunk 2 above)
Comment (HenryChen666) [reply src/app/pages/my-review/my-review.component.ts:9]: Lol, I push to remote too early, fixed.

**Comment by HenryChen666** [comment]
> Good stuff, the only problem is that it would be better if this tab was on the user's dropdown or only when the user was logged in. Also, don't you think its more clean if we just call it "reviews"? Later on I can move the page to some user subdomain like `dcms.com/user/reviews` or `/my/reviews`

Good point, I have changed the subdomain name on my update. To your first point, I thought someone is doing the permission routing?

**Comment by seofernando25** [comment]
Ah yeah you're right 😅

I actually created a route for the patients but I forgot it wasn't on the main branch lol, my bad.
After I'm done with that I can move everything to the patient domain.

it would look kinda like this in the end: https://github.com/SeoFernando25/dcms-g6/commit/d1b48d3f6e05fd9f85dfde5a53dd4ade19241a42#diff-f3dd17e0991db9edff2b7f8efe1b499d794e0c5dbcb17bdbaaef2c0d34a18f83R7

For now I guess you don't need to stress about it
